src/pages/cexs/index.tsx

```diff
@@ -294,6 +294,38 @@ export async function getStaticProps() {
 			}
 		})
 	)
+	
+	async function fetchTVL() {
+		const tvls = [];
+	  
+		for (const platform of cexData) {
+		  const response = await fetch(`https://api.llama.fi/tvl/${platform.slug}`);
+		  const tvl = await response.json();
+		  tvls.push({
+			name: platform.name,
+			tvl: tvl.cleanAssets
```

Comment:
If I change this line to   tvl: tvl
I get 
![image](https://user-images.githubusercontent.com/93603962/209678543-6524e63e-c8d0-4ef9-a010-ba28f2697054.png)

Reply:
![image](https://user-images.githubusercontent.com/93603962/209683991-153d2e5f-ecc8-4967-a6b6-b601639be3fe.png)
Also need to include some code to exclude protocols that don't have TVL

Reply:
@Define101 I am not familiar with UI code, but I think you are refetching information, that we already have in line 253, maybe you can add line there to sort based on `cleanTvl` if that field is present